docs/workflows/bind-passkey.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Bind Passkey To User
+sidebar_position: 8
+
+# Display h2 to h2 headings
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import BackendGeneral from './\_bind-passkey-backend-general.mdx';
+import BackendReturn from './\_bind-passkey-backend-return.mdx';
+import BackendEmail from './\_bind-passkey-backend-email.mdx';
+import FrontEndReturn from './\_bind-passkey-frontend-return.mdx';
+import FrontEndEmail from './\_bind-passkey-frontend-email.mdx';
```

Comment:
This is not a bad idea. Maybe this is also worth doing with the SDK Setup, especially if we're maintaining the existing SDK Docs, though I might wait on #202